airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/kotlin/io/airbyte/integrations/base/destination/typing_deduping/CatalogParser.kt

```diff
@@ -161,6 +161,45 @@ constructor(
                 var i = 1
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 104]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->wise suggestion we should probably follow<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/36805?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
done in https://github.com/airbytehq/airbyte/pull/36805/commits/c6101974d452aa98e2376068351d37689b32c6eb?w=1. also cleaned up some of the kotlin autoconvert cruft.